ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/acm-group-du-sno-ranGen.yaml

```diff
@@ -69,126 +69,16 @@ policies:
-              logAnnounceInterval -3
-              logSyncInterval -4
-              logMinDelayReqInterval -4
-              logMinPdelayReqInterval -4
-              announceReceiptTimeout 3
-              syncReceiptTimeout 0
-              delayAsymmetry 0
-              fault_reset_interval -4
-              neighborPropDelayThresh 20000000
-              masterOnly 0
-              G.8275.portDS.localPriority 128
-              #
-              # Run time options
-              #
-              assume_two_step 0
-              logging_level 6
-              path_trace_enabled 0
-              follow_up_info 0
-              hybrid_e2e 0
-              inhibit_multicast_service 0
-              net_sync_monitor 0
-              tc_spanning_tree 0
-              tx_timestamp_timeout 50
-              unicast_listen 0
-              unicast_master_table 0
-              unicast_req_duration 3600
-              use_syslog 1
-              verbose 0
-              summary_interval 0
-              kernel_leap 1
-              check_fup_sync 0
-              clock_class_threshold 7
-              #
-              # Servo Options
-              #
-              pi_proportional_const 0.0
-              pi_integral_const 0.0
-              pi_proportional_scale 0.0
-              pi_proportional_exponent -0.3
-              pi_proportional_norm_max 0.7
-              pi_integral_scale 0.0
-              pi_integral_exponent 0.4
-              pi_integral_norm_max 0.3
-              step_threshold 2.0
-              first_step_threshold 0.00002
-              max_frequency 900000000
-              clock_servo pi
-              sanity_freq_limit 200000000
-              ntpshm_segment 0
-              #
-              # Transport options
-              #
-              transportSpecific 0x0
-              ptp_dst_mac 01:1B:19:00:00:00
-              p2p_dst_mac 01:80:C2:00:00:0E
-              udp_ttl 1
-              udp6_scope 0x0E
-              uds_address /var/run/ptp4l
-              #
-              # Default interface options
-              #
-              clock_type OC
-              network_transport L2
-              delay_mechanism E2E
-              time_stamping hardware
-              tsproc_mode filter
-              delay_filter moving_median
-              delay_filter_length 10
-              egressLatency 0
-              ingressLatency 0
-              boundary_clock_jbod 0
-              #
-              # Clock description
-              #
-              productDescription ;;
-              revisionData ;;
-              manufacturerIdentity 00:00:00
-              userDescription ;
-              timeSource 0xA0
+        - metadata:
+            name: du-ptp-slave
+          spec:
+            profile:
+                - interface: ens5f0
+                  name: slave
+                  phc2sysOpts: -a -r -n 24
+                  ptp4lOpts: -2 -s --summary_interval -4
+      openapi:
+        path: schema.openapi
```

Comment:
@edcdavid   Shouldn't we have a ptp specific file name? schema.openapi sounds a like a generic name, wheras the schema is based on ptp operator if i'm not mistaken.

Reply:
Actually, I named the schema.openapi file on purpose to be generic since the schema does not necessarily addresses a single area/resource. It could mix schemas for multiple resources (this is actually how it is retrieved from a running cluster for instance) and a single file with all schemas needed could be easier to manage. If you feel strongly about it I can rename it.

Reply:
What is the recommendation when there are multiple different schema? Should we keep in a single file, just trying to understand the best practice.

Reply:
yes, that would be my recommendation. Keep a single file for all required schemas for a given repo and pass the same file to all patches that require it.